docker-storage-setup.sh

```diff
@@ -578,6 +578,11 @@ EOF
     if ! wait_for_dev ${dev}1; then
       Fatal "Partition device ${dev}1 is not available"
     fi
+    # Sometimes there is still a old filesystem signature where we
+    # have just created our new partition.  Pvcreate will find it and
+    # ask for confirmation to overwrite it.  So we wipe the new
+    # partition first.
+    wipefs -a ${dev}1
```

Comment:
> So I like the idea of bailing out if disk has fs signatures present and user can run wipefs before passing that disk to dss.

For me, the confusing bit is that the signature is on a partition that d-s-s has just created itself.  The disk didn't have any partition table on it when I gave it to d-s-s.

It appears to me to be quite tricky to know in what way d-s-s wants its disks to be wiped if it reserves the right to reject a disk as soon as it finds a tiny hint of old content.